resources/assets/js/videos/components/videoScreen.vue

```diff
@@ -77,16 +77,30 @@
                     @click="drawPoint"
                     >
                         <control-button
+                            v-if="this.singleAnnotation"
+                            icon="fa-check"
+                            title="Disable the single-frame annotation option to create multi-frame annotations"
+                            :disabled="true"
+                            ></control-button> 
+                        <control-button
+                            v-else
                             icon="fa-check"
                             title="Finish the point annotation 𝗘𝗻𝘁𝗲𝗿"
-                            :disabled="cantFinishDrawAnnotation"
+                            :disabled="this.cantFinishDrawAnnotation"
```

Comment:
Please don't use `this` anywhere in this template. It works without, too: `v-if="cantFinishDrawAnnotation"`.